src/scripts/lightgbm_python/score.py

```diff
@@ -95,6 +97,21 @@ def run(args, other_args=[]):
     with metrics_logger.log_time_block("inferencing"):
         booster.predict(data=inference_raw_data, predict_disable_shape_check=bool(args.predict_disable_shape_check))
 
+    import pandas as pd
```

Comment:
Let's split each framework into a distinct script. This way, we can maintian a collection of independent scripts, each reporting their metrics.

Can you create your score.py under the directory src/scripts/treelite_python/ instead?